tests/netlify-deploy.ts

```diff
@@ -142,17 +133,29 @@ export class NextDeployInstance extends NextInstance {
 
     if (deployRes.exitCode !== 0) {
       throw new Error(
-        `Failed to deploy project ${deployRes.stdout} ${deployRes.stderr} (${deployRes.exitCode})`,
+        `Failed to deploy project (${deployRes.exitCode}) ${deployRes.stdout} ${deployRes.stderr} `,
       )
     }
 
     try {
-      const data: NetlifyDeployResponse = JSON.parse(deployRes.stdout)
-      this._url = data.deploy_url
+      const [url] = new RegExp(/https:.+\.netlify\.app/gm).exec(deployRes.stdout) || []
+      if (!url) {
+        throw new Error('Could not extract the URL from the build logs')
+      }
+      const [deployID] = new URL(url).host.split('--')
+      this._url = url
       this._parsedUrl = new URL(this._url)
-      this._deployId = data.deploy_id
-      require('console').log(`Deployment URL: ${data.deploy_url}`)
-      require('console').log(`Logs: ${data.logs}`)
+      this._deployId = deployID
+      this._cliOutput = deployRes.stdout + deployRes.stdout
```

Comment:
🙃 is this a typo? did you mean stdout + stderr?

Reply:
https://github.com/opennextjs/opennextjs-netlify/pull/2698/commits/7faa8763172a6aad97225872879e6b4d9112f4fb